<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSelection</class>
 <widget class="QMainWindow" name="FileSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ã–sophagus Visualisierung</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>Previous Therapies</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="5" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="5" column="1">
            <widget class="QSpinBox" name="poem_spin"/>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="pneu_dil_40_spin"/>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="other_spin"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Times applied</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QDateEdit" name="pneu_dil_30_calendar">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QDateEdit" name="botox_calendar">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="pneu_dil_30_spin"/>
           </item>
           <item row="4" column="2">
            <widget class="QDateEdit" name="pneu_dil_40_calendar">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Last applied</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="other_check">
             <property name="text">
              <string>other</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="botox_check">
             <property name="text">
              <string>Botox</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Therapy</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="lapro_spin"/>
           </item>
           <item row="6" column="2">
            <widget class="QDateEdit" name="lapro_calendar">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="pneu_dil_35_check">
             <property name="text">
              <string>Pneumatische Dilatation 35ml</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="pneu_dil_35_spin"/>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="botox_spin"/>
           </item>
           <item row="3" column="2">
            <widget class="QDateEdit" name="pneu_dil_35_calendar">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="pneu_dil_30_check">
             <property name="text">
              <string>Pneumatische Dilatation 30ml</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="poem_check">
             <property name="text">
              <string>POEM</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="lapro_check">
             <property name="text">
              <string>Laparoskopische Myotomie</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="pneu_dil_40_check">
             <property name="text">
              <string>Pneumatische Dilatation 40ml</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QDateEdit" name="other_calendar">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QDateEdit" name="peom_calendar">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QCheckBox" name="botox_date_check">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QCheckBox" name="pneu_dil_35_date_check">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QCheckBox" name="pneu_dil_40_date_check">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QCheckBox" name="poem_date_check">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QCheckBox" name="lapro_date_check">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QCheckBox" name="other_date_check">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QCheckBox" name="pneu_dil_30_date_check">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Date unknown</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="save_button">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
